utils/tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-{flake8,unit}
+    py{27,35}-{flake8,unit,pylint}
```

Comment:
I tested out re-using virtualenvs between envs and it causes issues when using detox to run tests in parallel. I think the benefits of the parallel runs outweighs the drawbacks of the extra virtualenvs.

I'm good with consolidating the envs to have a single test run that runs multiple tests, but then that is a feature regression from the existing Makefile.